pkg/text/secretstore.go

```diff
@@ -0,0 +1,66 @@
+package text
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+
+	"github.com/fastly/go-fastly/v7/fastly"
+	"github.com/segmentio/textio"
+)
+
+func PrintSecretStoresTbl(out io.Writer, stores *fastly.SecretStores) {
```

Comment:
CI failed because of a missing comment on the exported type.

```suggestion
// PrintSecretStoresTbl displays store data in a table format.
func PrintSecretStoresTbl(out io.Writer, stores *fastly.SecretStores) {
```